strings/src/main/res/values/strings.xml

```diff
@@ -1252,6 +1252,9 @@
     <string name="use_mlkit_for_barcode_scanning">Use ML Kit for barcode scanning</string>
     <string name="mlkit_is_always_used_for_scanning_project_qr_codes">ML Kit is always used for scanning project QR codes</string>
 
+    <!-- Experimental setting that allows the number of repeated scans required for MLKit to process a barcode to be tweaked -->
+    <string name="mlkit_scan_threshold">MLKit scan threshold</string>
```

Comment:
As in other strings, this should be `ML Kit`, not `MLKit`. I can update it as @seadowg, you are already off till August 18th.